src/ModalWithFooter.scss

```diff
@@ -0,0 +1,41 @@
+@import 'variables';
+
+.ModalWithFooter__footer {
+    display: flex;
+    column-gap: 1rem;
+    padding: 1.5rem 1rem 1.5rem 0;
+    justify-content: flex-end;
+}
+
+button.btn.ModalWithFooter__secondaryButton,
+button.btn.ModalWithFooter__primaryButton {
+    border: 0;
+    padding: 0.2rem;
+    width: 8rem;
+
+    &:active {
+        background-color: #ED972C;
+    }
+
+    &:hover {
+        background-color: #F5C58A;
+        color: black;
+    }
+
+    &:focus {
+        box-shadow:
+            0 0 0 0.1rem #D5D8DD,
+            0 0 0 $c2lc-focus-indicator-width black;
+    }
+}
+
+button.btn.ModalWithFooter__secondaryButton {
+    margin-left: 5rem;
```

Comment:
What is the purpose of this `margin-left` on the secondary button?

Reply:
The new modal design has some spacing left to the cancel buttons, but I see this can be an issue when there's only one Primary button, or multiple secondary buttons. I think putting padding on the footer container makes more sense.